payroll-processor-client/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "npm run ng serve -- --aot --port 4201",
+    "start": "npm run ng serve -- --aot --port 4201 --configuration local",
```

Comment:
There's also a shorthand for this flag - `-c local`

Reply:
updated!